ctapipe/core/tool.py

```diff
@@ -109,21 +109,38 @@ def main():
             "command-line parameters"
         ),
     ).tag(config=True)
-    log_format = Unicode(
-        "%(levelname)s [%(name)s] (%(module)s/%(funcName)s): %(message)s",
-        help="The Logging format template",
-    ).tag(config=True)
 
-    _log_formatter_cls = ColoredFormatter
+    log_config = Path(
+        default_value=None, exists=True, directory_ok=False,
+        help="Filename of Logging Configuration",
+    ).tag(config=True)
+    log_level = Enum(
+        values=Application.log_level.values,
+        default_value="INFO",
+        help="Logging Level for Console Logging",
+    ).tag(config=True)
+    log_file = Path(
+        default_value=None, exists=None, directory_ok=False,
+        help="Filename for the log",
+    ).tag(config=True)
+    log_file_level = Enum(
+        values=Application.log_level.values,
+        default_value="DEBUG",
```

Comment:
I would not put DEBUG here. Info should be fine as default. 

Probably `WARNING` for `log_level` and `INFO` for `log_file_level` would be a sane default.